dev-tools/ecs-migration.yml

```diff
@@ -90,3 +90,33 @@
   to: source.domain
   alias: true
   copy_to: false
+
+# Filebeat modules
+
+## Apache module
+
+### Error fileset
+- from: apache.error.message
+  to: message
+  alias: true
+  copy_to: false
+
+- from: apache.error.level
+  to: log.level
+  alias: true
+  copy_to: false
+
+- from: apache2.error.client
+  to: source.ip
+  alias: true
+  copy_to: false
+
+- from: apache2.error.pid
+  to: process.pid
+  alias: true
+  copy_to: false
+
+- from: apache2.error.tid
+  to: process.tid
```

Comment:
Threads can also have a name, a priority, and perhaps even other properties. I would name this field `process.thread.tid`, to make space for other attributes we may want to eventually support about threads...

Reply:
Consider this:

```YAML
process.thread.tid
process.thread.name
process.thread.priority
```

I don't yet have a strong opinion on nesting under process vs top level. But I think it's really important we have a "thread" field set, so we can eventually track other things about threads.

Reply:
As a thread always belongs to a thread (is that correct 🤔 ) I think it makes sense to nest it under process. Should we open an issue / PR in ECS?

Reply:
Agreed. That's what I've been dying to do. Open this PR and about 15 others ;-)